### PR TITLE
Feature/dockstore work

### PR DESCRIPTION
**Pull request by denis-yuen**
- reflects update to dockstore and update readme path

### CODE REVIEW

**Comment by denis-yuen** [comment]
@tetron  FYI, heads-up

File: ga4gh_tool_registry/validate.py [hunk 1]
```diff
@@ -7,41 +7,20 @@
...
 from . import swg2salad
 import ruamel.yaml as yaml
+import warnings as warnings
 import requests
 from schema_salad.schema import load_schema, validate_doc
 from schema_salad import jsonld_context
 from schema_salad.main import printrdf
+from schema_salad.sourceline import cmap
 from schema_salad.ref_resolver import Loader
+from ruamel.yaml.comments import CommentedSeq, CommentedMap
```
Comment (garyluu) [inline ga4gh_tool_registry/validate.py:17]: Is this used anywhere?

File: ga4gh_tool_registry/validate.py [hunk 2]
```diff
@@ -7,41 +7,20 @@
...
-            else:
-                r['descriptor'] = ''
-
-        if 'tooltype' in r:
-            r["toolclass"] = r['tooltype']
-            del r['tooltype']
-
-        for d in r.values():
-            dockstore_fixup(d)
-
```
Comment (garyluu) [inline ga4gh_tool_registry/validate.py]: Not really important but PEP 8 expect 2 blank lines.  found 1

File: ga4gh_tool_registry/validate.py (hunk 1 above)
Comment (denis-yuen) [reply ga4gh_tool_registry/validate.py:17]: https://github.com/ga4gh/tool-registry-validator/pull/1/files#diff-9a53250075f7b337251e785447ce0c7fR47 I think

File: ga4gh_tool_registry/validate.py (hunk 2 above)
Comment (denis-yuen) [reply ga4gh_tool_registry/validate.py]: I think we can probably fix this when we get CI running on this repo, thx!

Comment (denis-yuen) [reply ga4gh_tool_registry/validate.py]: Actually, @garyluu  do you mind fixing this and adding a rudimentary travis.ci for running PEP? Thanks!

**Review by garyluu — APPROVED** [review]

**Pull request merged**